providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -37,6 +37,16 @@ static OSSL_OP_asym_cipher_gettable_ctx_params_fn rsa_gettable_ctx_params;
 static OSSL_OP_asym_cipher_set_ctx_params_fn rsa_set_ctx_params;
 static OSSL_OP_asym_cipher_settable_ctx_params_fn rsa_settable_ctx_params;
 
+static OSSL_ITEM padding_item[] = {
+    { RSA_PKCS1_PADDING,        "pkcs1"  },
+    { RSA_SSLV23_PADDING,       "sslv23" },
+    { RSA_NO_PADDING,           "none"   },
+    { RSA_PKCS1_OAEP_PADDING,   "oaep"   }, /* Correct spelling first */
+    { RSA_PKCS1_OAEP_PADDING,   "oeap"   },
```

Comment:
dooh!

Reply:
Not sure if that was just an exclamation or if you're trying to say something...  anyway, this is copied from existing legacy code, so it's obviously there for a reason...

https://github.com/openssl/openssl/blob/b0031e5dc2c8c99a6c04bc7625aa00d3d20a59a5/crypto/rsa/rsa_pmeth.c#L597-L619